test/mock/rtps/RTPSParticipantAttributes/fastdds/rtps/attributes/RTPSParticipantAttributes.hpp

```diff
@@ -16,8 +16,8 @@
  * @file RTPSParticipantAttributes.hpp
  */
 
-#ifndef FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
-#define FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
+#ifndef FASTDDS_RTPS_ATTRIBUTES__RTPSParticipantAttributes_HPP
+#define FASTDDS_RTPS_ATTRIBUTES__RTPSParticipantAttributes_HPP
```

Comment:
I would leave them as they were
```suggestion
#ifndef FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
#define FASTDDS_RTPS_ATTRIBUTES__RTPSPARTICIPANTATTRIBUTES_HPP
```

Reply:
I think it makes change to change, so that it conforms the style of all the guards

Reply:
Sorry I overlooked this